__mocks__/fileMock.js

```diff
@@ -1 +1 @@
-module.exports = 'test-file-stub';
\ No newline at end of file
+module.exports = "test-file-stub";
```

Comment:
The double quote changes in all build files is a remanent of your prior changes to eslint... if you're not going to include the ability to alter these files in the NPM `eslint` script you may want to revert all single to double quote updates

Reply:
ah, I thought I had fixed that, sorry. I guess we should only bother with linting stuff in `src/` anyway

Reply:
Thanks @cdcabrera. Pushed a version that reverts style changes to all files outside `./src`, so I'm only changing things affected by the lint config changes. These were indeed remnants of when I was using the prettier CLI directly.